exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/serialization/TransactionResultSerializer.java

```diff
@@ -16,13 +16,18 @@
 
 package com.exonum.binding.blockchain.serialization;
 
+import static com.exonum.binding.blockchain.TransactionResult.MAX_USER_DEFINED_ERROR_CODE;
+
 import com.exonum.binding.blockchain.TransactionResult;
 import com.exonum.binding.common.serialization.Serializer;
 import com.google.protobuf.InvalidProtocolBufferException;
 
 public enum TransactionResultSerializer implements Serializer<TransactionResult> {
   INSTANCE;
 
+  private static final int SUCCESSFUL_RESULT_STATUS_CODE = 256;
```

Comment:
I'd probably keep as is, because arithmetic operations on error codes don't make much sense.

Reply:
I don't like arithmetic operations on error codes either (and some other code style features in our core module :) ). But the code as it is right now seems a bit messy, as we defined one constant in one class and two others in another. Maybe it would make sense to define all of them in `TransactionResult` class? That way we could avoid arithmetic operations, but they would be stored in the same place and relationship between them would be more clear.

Reply:
Putting in one place is definitely better, `TxResult` is a good place)